packages/web/src/components/play-bar/desktop/PlayBar.js

```diff
@@ -350,6 +354,10 @@ class PlayBar extends Component {
     const favoriteText = favorited ? messages.unfavorite : messages.favorite
     const repostText = reposted ? messages.reposted : messages.repost
     const matrix = isMatrix()
+    const isPodcast = track?.genre === Genre.PODCASTS ?? false
```

Comment:
i think this is fine, but you i don't think you technically need the ?? false

Reply:
yea i think youre right, will change